### DISCUSSION TITLE
Retroalimentación de práctica del examen parcial #1 del módulo 3 (202201/CEM/[401, 401])

### DISCUSSION THREAD

**Discussion by marcianomoreno** [naylalex/tec-tc2005b-students#2] [Q&A]
Agradezco sus comentarios respecto a la práctica del examen parcial que efectuamos la semana del 14 de marzo de 2022:
1. Claridad de las instrucciones del examen de práctica.
2. Facilidad de uso de las herramientas tecnológicas involucradas (GitHub Codespaces).
3. Dudas de conceptos incluidos en el examen de práctica.
4. Otros comentarios y sugerencias.

**Comment by gggandre** [comment]
1. Claridad de las instrucciones del examen de práctica.
Se específica bien en el readme la instrucciones y lo que se solicita.
2.Facilidad de uso de las herramientas tecnológicas involucradas (GitHub Codespaces).
Es una buena herramienta para trabajar, ya que es desde el navegador, me parece bien.
3. Dudas de conceptos incluidos en el examen de práctica.
Solo saber si puede subir la solución final, ya que me quedé en lo último que hizo usted pero aún no me dice si la palabra es palindromo o no
4.Otros comentarios y sugerencias.

**Reply by marcianomoreno** [reply]
Gracias por tus comentarios @gggandre. Publicaré un tutorial para que puedan implementar la función isPalindrome(),

**Comment by A01745158** [comment]
1. Claridad de las instrucciones del examen de práctica. ---> Respecto a este punto considero que hicieron falta que unos puntos fueran detallados explícitamente 
2. Facilidad de uso de las herramientas tecnológicas involucradas (GitHub Codespaces). --> Me parece una excelente herramienta sin embargo el único inconveniente es tener que descargar por ejemplo Live Server además de configurar el modo en lo que lo queremos ver
3. Dudas de conceptos incluidos en el examen de práctica. --> 
4. Otros comentarios y sugerencias. --> Me hubiera gustado haber practicado más a lo largo de las clases.

**Reply by marcianomoreno** [reply]
Gracias @A01745158 

1. Por favor dime qué puntos requieren de más claridad.
2. Consulta la siguiente nota del Wiki para sincronizar tus settings de Visual Studio Code (incluyendo Live Server) [Sincronización de configuración en Visual Studio Code (Web)](https://github.com/naylalex/tec-tc2005b-students/wiki/Sincronizaci%C3%B3n-de-configuraci%C3%B3n-en-Visual-Studio-Code-(Web))
3. Haremos más prácticas, balanceando el tiempo de exposición de conceptos y prácticas fuera de las horas de clase.

**Comment by pablogonzalezdelaparra** [comment]
Claridad de las instrucciones del examen de práctica -> Las instrucciones son bastantes claras a lo largo de la práctica. Quizás una pequeña explicación de la función básica de ciertos pasos sería bastante beneficioso
Facilidad de uso de las herramientas tecnológicas involucradas (GitHub Codespaces). -> Considero que el Github Codespace es bastante accesible y sencillo de utilizar. Ayuda mucho con las acciones de Git.
Dudas de conceptos incluidos en el examen de práctica. -> Un poco acerca del webpack y sus características más importantes/notables en la resolución (en este caso) de la práctica.
Otros comentarios y sugerencias. -> Quizás una sección tipo "Troubleshooting" para resolver los problemas más comunes que puedan llegar a ocurrir en el proceso.
Quizás la realización de una pequeña guía que sintetice (o resuma) cual es la relación de los archivos tanto en su construcción como en su ejecución (Como se relacionan el uno con el otro).

**Reply by marcianomoreno** [reply]
Gracias @pablogonzalezdelaparra 

1. Revisaremos los detalles de los comandos cuando presente el tema de Node.Js.
2. Proveeré detalles adicionales de webpack la semana entrante.
3. Por favor dime algunos problemas de interés para crear una sección de troubleshooting en nuestro wiki.
4. OK, entiendo que es el tema de webpack y cómo se inyecta contenido.

**Comment by a01745865** [comment]
1. Claridad de las instrucciones del examen de práctica.

Considero que las instrucciones estan bien detalladas sin embargo llega a haber pasos que se omiten y llega a causar contratiempos en identificar que se tiene que hacer en esos pasos. 

2. Facilidad de uso de las herramientas tecnológicas involucradas (GitHub Codespaces).

Codespace me pareció bastante sencilla de ocupar.

3. Dudas de conceptos incluidos en el examen de práctica.

Considero que estaría bien que se definiera un poquito mejor el tema de la instalacion de webpack y los parámetros que se le tienen que mandar en los archivos de tscconfig

Otros comentarios y sugerencias.
Si me gustaría un poco mas de detalle en cada comando que se le manda y en que ocasiones se ocupa para poder saber que comandos tendriamos que usar en dado caso de querer crear un proyecto por nuestra cuenta.

**Reply by marcianomoreno** [reply]
Gracias @a01745865 
En [Issue 6](https://github.com/naylalex/tec-tc2005b-students/issues/6) daré seguimiento a los temas de webpack que mencionas.

Por favor dime los comandos en los que tienes más dudas.
